### PR TITLE
Port not warmed-up configurations log warnings

### PR DESCRIPTION
**Pull request by adobrzyniewicz-habana**
From: https://github.com/HabanaAI/vllm-fork/pull/94

### CODE REVIEW

File: vllm/worker/habana_model_runner.py [hunk 1]
```diff
@@ -1604,10 +1615,7 @@ def execute_model(
         if multi_modal_input is not None:
             execute_model_kwargs.update(multi_modal_input)
         if htorch.utils.internal.is_lazy():
-            execute_model_kwargs.update({
-                "bypass_hpu_graphs": not use_graphs,
-                "warmup_mode": warmup_mode
-            })
+            execute_model_kwargs.update({"bypass_hpu_graphs": not use_graphs})
```
Comment (kzawora-intel) [inline vllm/worker/habana_model_runner.py:1618]: why is warmup_mode kwarg being removed here?

Comment (adobrzyniewicz-habana) [reply vllm/worker/habana_model_runner.py:1618]: It was also removed in original PR - should we leave that?

Comment (adobrzyniewicz-habana) [reply vllm/worker/habana_model_runner.py]: Reverted

**Review by kzawora-intel — APPROVED** [review]
lgtm

**Pull request merged**